.vscode/launch.json

```diff
@@ -17,11 +17,11 @@
             "name": "Remote",
             "restart": false,
             "protocol": "inspector",
-            "port": 9222,
+            "port": 9229,
```

Comment:
did changing this resolve a debugging issue?

Reply:
No. Still not able to hook VCCode debugger with dockerise port.